### ISSUE TITLE
Ivy-hydra menu is outdated 

### ISSUE THREAD

**Issue by dalanicolai** [syl20bnr/spacemacs#13361]
#### Description :octocat:
Ivy-hydra menu outdated (e.g. no mark option available)

#### Reproduction guide :beetle:
- Start Emacs
- install ivy layer
- open ivy hydra (e.g. SPC f f and Ctrl-o on some file)

*Observed behaviour:* :eyes: :broken_heart:
The hydra menu is spacemacs looks outdated. E.g. in the most recent hydra-menu, `matcher` is bound to "M" (instead of "m") while "m" is bound to `mark`. Also see the last part of section [5.2.6 of the Ivy User Manual](https://oremacs.com/swiper/#key-bindings), where it says 

> Hydra menu also offers bindings for marking multiple candidates:

These features seem unavailable in spacemacs.

I had a look if I could fix it but spacemacs seems to use the most recent Ivy and Ivy-hydra packages (20191018.1251, i.e. same as in emacs), and I could not find where the spacemacs hydra-menu is defined. I'll attach images of the current hydra-menu in emacs and in spacemacs.

![Screenshot from 2020-03-10 12-44-11](https://user-images.githubusercontent.com/18429791/76309530-3ed22a80-62cd-11ea-976e-cc418bd425a2.png)

![Screenshot from 2020-03-10 12-37-09](https://user-images.githubusercontent.com/18429791/76309539-42fe4800-62cd-11ea-9227-a5e94d126259.png)

*Expected behaviour:* :heart: :smile:
Spacemacs should show updated Ivy-hydra menu (see previous section)

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 884afe138)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: ivy
- Layers:
```elisp
(emacs-lisp ivy multiple-cursors treemacs)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND DBUS GSETTINGS GLIB NOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS LCMS2

#### Backtrace :paw_prints:
```
<<BACKTRACE IF RELEVANT>>
```

**Comment by dalanicolai** [comment]
So I found that the overwriting keymap is defined in packages.el of the spacemacs-completion layer.
Just before where the overwriting keymap is defined I find the following comment

>     ;; ivy-hydra disabled for now, waiting to see how the dependency management
>    ;; evolves upstream
>    ;; (require 'ivy-hydra)))

So just uncommenting that require function and commenting out the keymap seems to work perfectly fine so that Ivy just shows its own (updated) keymap and correct functionality

**Comment by smile13241324** [comment]
@dalanicolai I have just asked the other maintainer why this is deactivated, feel free to provide a PR in the meantime and see whether there are any side effects of this. If no problems arise and nobody objects we can just merge your PR and ivy is up-to-date again.

**Comment by dalanicolai** [comment]
Yes fine, should I create a new branch or can I create the PR directly to the develop branch (because I have it ready like that, but then I found maybe I should create separate branch first. Sorry but I am learning git)

**Comment by smile13241324** [comment]
Best make a new branch but you can do that directly from your modified develop branch. There is a good tutorial now in the CONTRIBUTION.org file how to make a PR I think the author uses maggit which is really a great tool to work with git and does support all features you can do from the terminal.

Have fun with git, I have learned it years ago coming from SVN and never looked back :)

**Comment by smile13241324** [comment]
I have merged your PR #13377 please close this issue if the menu is ok now.

**Comment by nixmaniack** [comment]
> I have just asked the other maintainer why this is deactivated

For the posterity, here's the dependency discussion https://github.com/abo-abo/swiper/issues/464 that code comment mentions which was resolved but Spacemacs wasn't updated since then.
